vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -3,7 +3,7 @@ function update_dashboard_theme(theme_selector_checked) {
     "data-bs-theme",
     theme_selector_checked ? "light" : "dark",
   );
-  return theme_selector_checked ? "vizro_light" : "vizro_dark";
+  return window.dash_clientside.no_update;
```

Comment:
I guess we can remove `return` line from here all together with the `dash.Output` from the `update_dashboard_theme` `clientside_callback` in the `Dashboard.build`.  
Reference -> https://github.com/plotly/dash/pull/2867

Reply:
Done! :) Thank you!